static/bidder-info/tgm.yaml

```diff
@@ -0,0 +1 @@
+aliasOf: "limelightDigital"
```

Comment:
Could you please provide a reason for creating this alias and not use limelightDigital adapter directly?

Reply:
It's our client's preference, they would like to have their own named adapter rather than use `limelightDigital`